test/unit/spec/v1/EnrollSms_spec.js

```diff
@@ -31,8 +31,8 @@ Expect.describe('EnrollSms', function() {
       el: $sandbox,
       baseUrl: baseUrl,
       authClient: authClient,
-      defaultCountryCode: routerOptions.defaultCountryCode,
```

Comment:
Why did this get removed?

Reply:
In line 35 I added 
https://github.com/okta/okta-signin-widget/blob/62ef3c5118b26826655fb35f70bf3aa2a1d84391/test/unit/spec/v1/EnrollSms_spec.js#L35
to destructure all properties from `routerOptions` including `defaultCountryCode ` and also `i18n`